client/app/email-domain/email/account/migrate/email-domain-email-account-migrate.html

```diff
@@ -124,55 +124,64 @@
-                        <div class="col-md-8">
-                            <p class="form-control-static" data-ng-bind="ctrl.migrate.serviceType"></p>
-                        </div>
+                </div>
+                <div class="form-group mb-0">
+                    <label class="control-label col-md-4" data-i18n-static="email_tab_modal_migrate_service_type_label"></label>
+                    <div class="col-md-8">
+                        <p class="form-control-static" data-ng-bind="ctrl.migrate.serviceType"></p>
                     </div>
-                    <div class="form-group">
-                        <label class="control-label col-md-4" data-i18n-static="email_tab_modal_migrate_destination_service_label"></label>
-                        <div class="col-md-8">
-                            <p class="form-control-static" data-ng-bind="ctrl.migrate.destinationService"></p>
-                        </div>
+                </div>
+                <div class="form-group">
+                    <label class="control-label col-md-4" data-i18n-static="email_tab_modal_migrate_destination_service_label"></label>
+                    <div class="col-md-8">
+                        <p class="form-control-static" data-ng-bind="ctrl.migrate.destinationService"></p>
                     </div>
-                    <!-- /Migration Data -->
-
                 </div>
+                <!-- /Migration Data -->
 
-                <!-- Guides Text -->
-                <p data-ng-if="ctrl.migrate.serviceType === 'EMAIL PRO'" data-i18n-static="email_tab_modal_migrate_configuration_guides_emailpro"></p>
-                <p data-ng-if="ctrl.migrate.serviceType === 'HOSTED EXCHANGE'" data-i18n-static="email_tab_modal_migrate_configuration_guides_exchange"></p>
-                <p>
-                    <a class="btn btn-default btn-sm" href="https://docs.ovh.com/fr/fr/web/" target="_blank">
-                        <span data-i18n-static="email_tab_modal_migrate_configuration_guides_button"></span>
-                        <span class="fa fa-external-link"></span>
-                    </a>
-                </p>
-                <!-- /Guides Text -->
-
-                <!-- Not Supported Alert -->
-                <div class="alert alert-warning" role="alert">
-                    <p data-ng-bind-html="tr('email_tab_modal_migrate_alert_not_supported')"></p>
-                </div>
-                <!-- /Not Supported Alert -->
+            </div>
+
+            <!-- Guides Text -->
+            <p data-ng-if="ctrl.migrate.serviceType === 'EMAIL PRO'" data-i18n-static="email_tab_modal_migrate_configuration_guides_emailpro"></p>
+            <p data-ng-if="ctrl.migrate.serviceType === 'HOSTED EXCHANGE'" data-i18n-static="email_tab_modal_migrate_configuration_guides_exchange"></p>
+            <p>
+                <a class="btn btn-default btn-sm" href="https://docs.ovh.com/fr/fr/web/" target="_blank">
```

Comment:
This URL doesn't exist in constant?

Reply:
Didn't know, but yes it is. I'll change it.